Marlin/src/config/examples/wt150/Configuration.h

```diff
@@ -1559,16 +1559,24 @@
  * Adds the M150 command to set the LED (or LED strip) color.
  * If pins are PWM capable (e.g., 4, 5, 6, 11) then a range of
  * luminance values can be set from 0 to 255.
+ * For Neopixel LED overall brightness parameters is also available
  *
  * *** CAUTION ***
  *  LED Strips require a MOFSET Chip between PWM lines and LEDs,
  *  as the Arduino cannot handle the current the LEDs will require.
  *  Failure to follow this precaution can destroy your Arduino!
+ *  Neopixel LED is 5V powered, but linear 5V regulator on Arduino
+ *  cannot handle the current, separate 5V power supply must be used
  * *** CAUTION ***
  *
+ * LED type. This options are mutualy exclusive. Uncomment only one.
```

Comment:
Actually, `NEOPIXEL_LED` can operate in conjunction with `RGB_LED` or `RGBW_LED`. The definition should go below where the previous definition was.